docs/presto-admin-commands.rst

```diff
@@ -113,7 +113,7 @@ configurations:
 *jvm.config* ::
 
     -server
-    -Xmx2G
+    -Xmx16G
```

Comment:
Should we correspondingly also increase the values for query.max_memory_per_node?